specs/core/1_shard-data-chains.md

```diff
@@ -327,53 +314,33 @@ def is_valid_shard_block(beacon_blocks: Sequence[BeaconBlock],
         assert signing_root(beacon_blocks[parent_block.slot]) == parent_block.beacon_chain_root
 
     # Check attestations
-    assert len(candidate.attestations) <= MAX_SHARD_ATTESTIONS
-    for _, attestation in enumerate(candidate.attestations):
-        assert max(GENESIS_SHARD_SLOT, candidate.slot - SLOTS_PER_EPOCH) <= attestation.data.slot
-        assert attestation.data.slot <= candidate.slot - MIN_ATTESTATION_INCLUSION_DELAY
-        assert attestation.data.crosslink.shard == candidate.shard
-        verify_shard_attestation_signature(beacon_state, attestation)
-
-    # Check signature
-    proposer_index = get_shard_proposer_index(beacon_state, candidate.shard, candidate.slot)
+    persistent_committee = get_persistent_committee(beacon_state, block.shard, block.slot)
+    pubkeys = []
+    for i, index in enumerate(persistent_committee):
```

Comment:
This is pulling from the entire persistent committee but the `attester_bitfield` is of size `SHARD_SLOT_COMMITTEE_SIZE`. 

This needs to be patched with some of the changes from #1276 to be complete/correct (specifically need `get_shard_block_attester_committee`)